src/InteractsWithMedia.php

```diff
@@ -289,24 +289,32 @@ public function getMediaModel(): string
         return config('media-library.media_model');
     }
 
+    private function getMediaItem(string $collectionName, $filters, string $firstOrLast)
```

Comment:
Make this one protected, and move this under the last function in this file that uses this.